plot.py

```diff
@@ -54,7 +56,7 @@ def plotDataMC(args,plot):
 	for background in backgrounds:
 		if args.use2016:
 			if background == "Jets":
-				processes.append(Process(getattr(Backgrounds2016,background),eventCounts,negWeights,normalized=True))
+				processes.append(Process(getattr(Backgrounds,background),eventCounts,negWeights,normalized=True))
```

Comment:
Why did you change this?

Reply:
When I used Backgrounds2016, the 2016 Jets background is not showing....

Reply:
Mhh, ok. But we should try to understand and fix that properly. Can you dig a little bit more? If you have trouble finding the source of the issue, I can also have a look.

Reply:
Sure. I just digged a little bit, the jets 2016 background is read from file jets_muons_2016.root, which ranges from 0.5-3000.5 GeV. When we're trying to load jets_2016 background, the program will first call loadHistoFromFile at (https://github.com/JanFSchulte/CIAnalysis/blob/master/helpers.py#L274), then this function will load the histogram ("jets") from file ("jets_muons_2016.root") which is correct, but then it will try to rebin it at (https://github.com/JanFSchulte/CIAnalysis/blob/master/helpers.py#L106) with a new binning from (https://github.com/JanFSchulte/CIAnalysis/blob/master/helpers.py#L42). However this new binning ranges from 51 to 4000 which exceeds the boundary.... So i think that's why it doesn't work... In contrast, the 2017 jets background has the same structure but it ranges from 0-7000, so it can successfully pass the binning.